src/main/java/slimeknights/tconstruct/fluids/TinkerFluids.java

```diff
@@ -1,53 +1,32 @@
-  public static RegistryObject<FlowingFluid> blue_slime_fluid_flowing = FLUIDS.register("blue_slime_fluid_flowing", () -> new SlimeFluid.Flowing(TinkerFluids.blue_slime_fluid_properties));
-  public static RegistryObject<FlowingFluidBlock> blue_slime_fluid_block = BLOCKS.register("blue_slime_fluid_block", () -> new FlowingFluidBlock(blue_slime_fluid, Block.Properties.create(Material.WATER).doesNotBlockMovement().hardnessAndResistance(100.0F).noDrops()));
-  public static RegistryObject<Item> blue_slime_fluid_bucket = ITEMS.register("blue_slime_bucket", () -> new BucketItem(blue_slime_fluid, new Item.Properties().containerItem(Items.BUCKET).maxStackSize(1).group(ItemGroup.MISC)));
-  public static final SlimeFluid.Properties blue_slime_fluid_properties = new SlimeFluid.Properties(blue_slime_fluid, blue_slime_fluid_flowing, FluidAttributes.builder(FluidIcons.FLUID_STILL, FluidIcons.FLUID_FLOWING).color(0xef67f0f5).density(1500).viscosity(1500).temperature(310)).explosionResistance(100.0F).bucket(blue_slime_fluid_bucket).block(blue_slime_fluid_block);
-
-  public static RegistryObject<FlowingFluid> purple_slime_fluid = FLUIDS.register("purple_slime_fluid", () -> new SlimeFluid.Source(TinkerFluids.purple_slime_fluid_properties));
-  public static RegistryObject<FlowingFluid> purple_slime_fluid_flowing = FLUIDS.register("purple_slime_fluid_flowing", () -> new SlimeFluid.Flowing(TinkerFluids.purple_slime_fluid_properties));
-  public static RegistryObject<FlowingFluidBlock> purple_slime_fluid_block = BLOCKS.register("purple_slime_fluid_block", () -> new FlowingFluidBlock(purple_slime_fluid, Block.Properties.create(Material.WATER).doesNotBlockMovement().hardnessAndResistance(100.0F).noDrops()));
-  public static RegistryObject<Item> purple_slime_fluid_bucket = ITEMS.register("purple_slime_bucket", () -> new BucketItem(purple_slime_fluid, new Item.Properties().containerItem(Items.BUCKET).maxStackSize(1).group(ItemGroup.MISC)));
-  public static final SlimeFluid.Properties purple_slime_fluid_properties = new SlimeFluid.Properties(purple_slime_fluid, purple_slime_fluid_flowing, FluidAttributes.builder(FluidIcons.FLUID_STILL, FluidIcons.FLUID_FLOWING).color(0xefd236ff).density(1600).viscosity(1600).temperature(370)).explosionResistance(100.0F).bucket(purple_slime_fluid_bucket).block(purple_slime_fluid_block);
+  public static final FluidObject<ForgeFlowingFluid> blue_slime_fluid = FLUIDS.register("blue_slime_fluid", FluidAttributes.builder(FluidIcons.FLUID_STILL, FluidIcons.FLUID_FLOWING).color(0xef67f0f5).density(1500).viscosity(1500).temperature(310), Material.WATER);
+  public static final FluidObject<ForgeFlowingFluid> purple_slime_fluid = FLUIDS.register("purple_slime_fluid", FluidAttributes.builder(FluidIcons.FLUID_STILL, FluidIcons.FLUID_FLOWING).color(0xefd236ff).density(1600).viscosity(1600).temperature(370), Material.WATER);
```

Comment:
Personally, I think `blue_slime` and `purple_slime` are better names, the fluid is redundant when the type is fluid, as names do not clash between registries. It also means the bucket will generate as `blue_slime_bucket` and `purple_slime_bucket`.

For the renames, if we change the name as I suggested (same for purple):
* Fluids
  * `blue_slime_fluid` -> `blue_slime` and `purple_slime_fluid` -> `purple_slime`
  * `blue_slime_fluid_flowing` -> `flowing_blue_slime` and purple
* Blocks
  * `blue_slime_fluid_block` -> `blue_slime_fluid` (I suffix `_fluid` for the block form, but leave off block because its redundant)

Reply:
using 'blue_slime' causes a crash, at least with the fluid reg class you provided in the gist.

Reply:
https://gist.github.com/hanetzer/4e874113a3d365c4c1fcdc3ce19cdb2e << nothing
changed from what's in this pr aside from renaming the reg name "blue_slime_fluid"
to "blue_slime"

Reply:
Yeah. I think the gist missed the suffix `_fluid` fixed.